pkg/cluster/cluster.go

```diff
@@ -390,6 +396,135 @@ func (c *Cluster) createPod(members etcdutil.MemberSet, m *etcdutil.Member, stat
+				Name:      "sensu-block-all",
+				Namespace: metav1.NamespaceDefault,
+			},
+			Spec: networkingv1.NetworkPolicySpec{
+				PodSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"app": "sensu",
+					},
+				},
+				Ingress: []networkingv1.NetworkPolicyIngressRule{},
+			},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "sensu-api-pods",
+				Namespace: metav1.NamespaceDefault,
+			},
+			Spec: networkingv1.NetworkPolicySpec{
+				PodSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"app": "sensu",
+					},
+				},
+				Ingress: []networkingv1.NetworkPolicyIngressRule{
+					{
+						Ports: []networkingv1.NetworkPolicyPort{
+							{
+								Port: &intstr.IntOrString{Type: intstr.Int, IntVal: 3000},
+							},
+							{
+								Port: &intstr.IntOrString{Type: intstr.Int, IntVal: 8080},
+							},
+							{
+								Port: &intstr.IntOrString{Type: intstr.Int, IntVal: 8081},
+							},
+						},
+						From: []networkingv1.NetworkPolicyPeer{},
+					},
+				},
+			},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "sensu-operator-pods",
+				Namespace: metav1.NamespaceDefault,
+			},
+			Spec: networkingv1.NetworkPolicySpec{
+				PodSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"app": "sensu",
+					},
+				},
+				Ingress: []networkingv1.NetworkPolicyIngressRule{
+					{
+						Ports: []networkingv1.NetworkPolicyPort{
+							{
+								Port: &intstr.IntOrString{Type: intstr.Int, IntVal: 2379},
+							},
+							{
+								Port: &intstr.IntOrString{Type: intstr.Int, IntVal: 2380},
+							},
+						},
+						From: []networkingv1.NetworkPolicyPeer{
+							{
+								PodSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{
+										"name": "sensu-operator",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+		{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "sensu-cluster-pods",
+				Namespace: metav1.NamespaceDefault,
+			},
+			Spec: networkingv1.NetworkPolicySpec{
+				PodSelector: metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						"app": "sensu",
+					},
+				},
+				Ingress: []networkingv1.NetworkPolicyIngressRule{
+					{
+						Ports: []networkingv1.NetworkPolicyPort{
+							{
+								Port: &intstr.IntOrString{Type: intstr.Int, IntVal: 2379},
+							},
+							{
+								Port: &intstr.IntOrString{Type: intstr.Int, IntVal: 2380},
+							},
+						},
+						From: []networkingv1.NetworkPolicyPeer{
+							{
+								PodSelector: &metav1.LabelSelector{
+									MatchLabels: map[string]string{
+										"app": "sensu",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		},
+	}
+
+	c.logger.Infof("creating NetworkPolicy for cluster %s", c.cluster.Name)
+	for _, net := range netCases {
+		_, err := c.config.KubeCli.NetworkingV1().NetworkPolicies(c.cluster.Namespace).Get(net.Name, metav1.GetOptions{})
+		if err != nil {
```

Comment:
Is there a way to be sure it's a "not found" and not a different error? I think we should at least add a comment here like

```
// No NetworkPolicy for Sensu clusters was installed yet, let's do it now
```

Reply:
done